db/migrate/20240826121501_add_big_int_shelved_notifications_notification_id.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class AddBigIntShelvedNotificationsNotificationId < ActiveRecord::Migration[7.0]
+  def up
+    # Short-circuit if the table has been migrated already
+    result =
+      execute(
+        "SELECT data_type FROM information_schema.columns WHERE table_name = 'shelved_notifications' AND column_name = 'notification_id' LIMIT 1",
+      )
+    data_type = result[0]["data_type"]
+    return if data_type.downcase == "bigint"
+
```

Comment:
```suggestion
```

We don't need this as there is no chance that the `notification_id` column has been migrated to `bigint`.